src/acl.ts

```diff
@@ -742,7 +741,7 @@ class Acl extends AclRoleAccessorMethods {
         let results;
 
         if (resp.items) {
-          results = arrify(resp.items).map(this.makeAclObject_);
+          results = resp.items.map(this.makeAclObject_);
```

Comment:
Just want to check that resp.items is always an array?

Reply:
yep. an array or undefined. we already check for undefined above.